.platform/routes.yaml

```diff
@@ -94,6 +94,9 @@ https://{default}/:
       # Drupal 7 EOL
       "/frameworks/drupal7.*": { "to": "/guides/drupal9/deploy.html", "code": 301, "regexp": true }
 
+      # Removal of featured frameworks
+      "/frameworks/(?!deploy|drupal)(.*).html":  { "to": "/guides/$1/deploy.html", "code": 301, "regexp": true }
```

Comment:
The nested link doesn't work, but it's probably safe to discard?
Or add one additional rule?

https://docs.platform.sh/frameworks/ibexa/fastly.html -> https://pr-2212-imxy7bi-652soceglkw4u.eu-3.platformsh.site/guides/ibexa/fastly/deploy.html
it should point here:
https://pr-2212-imxy7bi-652soceglkw4u.eu-3.platformsh.site/guides/ibexa/fastly.html

Reply:
Of course there's a subpage creating an exception. :facepalm: I'll add another rule.